### PR TITLE
Merging DB migration back to initial.

### PR DESCRIPTION
**Pull request by Fosol**
Updating Import Tool to refresh token.

### CODE REVIEW

File: backend/dal/Migrations/PIMSContextModelSnapshot.cs [hunk 1]
```diff
@@ -937,9 +937,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("DATETIME2")
                         .HasDefaultValueSql("GETUTCDATE()");
 
-                    b.Property<bool>("IsDisabled")
```
Comment (devinleighsmith) [inline backend/dal/Migrations/PIMSContextModelSnapshot.cs]: Hmm... what happened here?

File: backend/tools/import/appsettings.json [hunk 2]
```diff
@@ -6,9 +6,13 @@
...
     }
   },
+  "Keycloak": {
+    "TokenUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/quartech/protocol/openid-connect/token",
+    "ClientId": "pims-app"
+  },
   "Api": {
-    "Url": "http://pims-dev.pathfinder.gov.bc.ca/api/tools/import/properties",
-    "Method": "POST"
+    "ImportUrl2": "http://pims-dev.pathfinder.gov.bc.ca/api/tools/import/properties",
```
Comment (devinleighsmith) [inline backend/tools/import/appsettings.json]: /nit naming

File: backend/dal/Migrations/PIMSContextModelSnapshot.cs (hunk 1 above)
Comment (Fosol) [reply backend/dal/Migrations/PIMSContextModelSnapshot.cs]: This is auto-generated.

File: backend/tools/import/appsettings.json (hunk 2 above)
Comment (Fosol) [reply backend/tools/import/appsettings.json]: Ah forgot I made that change to test something.  Thanks for catching it.

**Pull request merged**